src/clues.ts

```diff
@@ -1,4 +1,34 @@
-export default {
+export interface Rule {
+  /** The name of the rule */
+  heading: string;
+  /** Shortened URL to the rule's documentation. */
+  docs: string;
+  /** Recommendation on how to fix the rule. */
+  suggestion: (value?: number | string) => string;
+  /** Explanation of the rule and why it's important. */
+  rationale: string;
+  /** URLs to external sources to support the rule. */
+  source: string[];
+  /** Example of the correct usage. */
+  ok: string;
+  /** Example of the incorrect usage. */
+  notOk: string;
+  /** If available, URL to https://doublegreat.dev/listen/ test case to support the rule.  */
+  listen?: string;
+  /** If applicable, list of strings to test the rule against.  */
+  rules?: string[];
+  /** If applicable, a function to test the rule against the current `alt` value. */
+  fn?: (item: string, alt: string) => string | boolean;
+}
```

Comment:
💡 This interface defines the shape for each rule, including key names, type, and whether it's optional (`?`). As a result, this PR is able to remove some unit tests that TypeScript made redundant in clues.test.js. Ref 4419555.